docs/plugins/filters/oui.asciidoc

```diff
@@ -0,0 +1,71 @@
+:plugin: oui
+:type: filter
+:default_plugin: 0
```

Comment:
New plugin file. Do we want to link it from list of filters?

Reply:
++ on adding it